src/Configuration/test/SpringBootBase.Test/SpringBootConfigurationBuilderExtensionsTest.cs

```diff
@@ -32,5 +32,37 @@ public void AddSpringBootEnv_AddKeys()
             var value = config["foo:bar"];
             Assert.Equal("value", value);
         }
+
+        [Fact]
+        public void AddSpringBootCmd_ThrowsIfConfigBuilderNull()
+        {
+            // Arrange
+            IConfigurationBuilder configurationBuilder = new ConfigurationBuilder();
+
+            // Act and Assert
+            Assert.Throws<ArgumentNullException>(() => SpringBootConfigurationBuilderExtensions.AddSpringBootCmd(null, configurationBuilder.Build()));
+
+            Assert.Throws<ArgumentNullException>(() => SpringBootConfigurationBuilderExtensions.AddSpringBootCmd(configurationBuilder, null));
```

Comment:
```suggestion
            var ex = Assert.Throws<ArgumentNullException>(() => SpringBootConfigurationBuilderExtensions.AddSpringBootCmd(null, configurationBuilder.Build()));
            Assert.Equal("builder", ex.ParamName);
            var ex2 = Assert.Throws<ArgumentNullException>(() => SpringBootConfigurationBuilderExtensions.AddSpringBootCmd(configurationBuilder, null));
            Assert.Equal("configuration", ex.ParamName);
```

Without asserting on the param name we can't confirm which param was null